modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,302 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= ROSA AMIs for {product-title}
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+List of Regions:
+
+. us-east-1
+. us-east-2
+. us-west-1
+. us-west-2
+. af-south-1
+. ap-east-1
+. ap-south-2
+. ap-southeast-3
+. ap-southeast-4
+. ap-south-1
+. ap-northeast-3
+. ap-northeast-2
+. ap-southeast-1
+. ap-southeast-2
+. ap-northeast-1
+. ca-central-1
+. eu-central-1
+. eu-west-1
+. eu-west-2
+. eu-south-1
+. eu-west-3
+. eu-south-2
+. eu-north-1
+. eu-central-2
+. me-south-1
+. me-central-1
+. sa-east-1
+
+[id="rosa-classic-aws-rhcos-ami-415_{context}"]
+== {product-title} 4.15 AMIs
```

Comment:
I'd like to see this table of contents changed into the following:
- Red Hat OpenShift Service on AWS (ROSA) AMIs
  - ROSA Classic
    - Version 4.15 AMIs
    - Version 4.14 AMIs
  - ROSA HCP
    - Version 4.15 AMIs
    - Version 4.14 AMIs

We can obviously leave HCP out for the time being, but that's how I'd like to see the ToC look when it is done.